tests/ValueObject/Primitive/DateTest.php

```diff
@@ -2,8 +2,9 @@
 
 declare(strict_types=1);
 
-namespace CNastasi\DDD\ValueObject\Primitive;
+namespace CNastasi\DDD\Tests\ValueObject\Primitive;
```

Comment:
Li metto aposta nello stesso namespace, difficile che ci sia conflitto e vengono caricati soltanto in --dev

Reply:
Rimosso namespace Tests